EpiserverRedirects/DynamicData/IDynamicDataRedirectRuleMapper.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Forte.EpiserverRedirects.Model.RedirectRule;
+
+
+namespace Forte.EpiserverRedirects.DynamicData
+{
+    public interface IDynamicDataRedirectRuleMapper
```

Comment:
Why not make this generic `IRedirectRuleMapper` and implement it for both DDS and EF?

Reply:
because the universal mapper will have only common logic. Store-specific logic - creating instance of entity and assigning its ID will be inside repository. It means we have mapping logic in 2 places. Which is no longer SOLID. And Unit tests will be ugly.
I will make it and you check it in commit.